io.home-assistant.opening/dataset.tt

```diff
@@ -0,0 +1,83 @@
+dataset @io.home-assistant.opening language "en" {
+
+  program := now => @io.home-assistant.opening.state() => notify
+  #_[utterances=["is the opening open?",
+                 "check if the opening is open",
+                 "is the opening closed?",
+                 "check if the opening is closed",
+                 "check if the opening is open or closed"]];
```

Comment:
Ok this one is outright bad. We can't just put stuff like this in blindly (or worse, in front of mturk workers), it's not going to work.

Reply:
This one is for an opening binary sensor where the on state means open and off state means closed. Actually I had some difficulty coming up with the utterances also for the cold, heat, occupancy, presence, problem, safety sensors, where the current utterances all sound pretty weird.